.github/workflows/code.yml

```diff
@@ -89,74 +137,51 @@ jobs:
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: ghcr.io/${{ github.repository }}
+          images: ${{ env.IMAGE_REPOSITORY }}
           tags: |
-            type=ref,event=branch
             type=ref,event=tag
+            type=raw,value=latest
 
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Build developer image for testing
-        uses: docker/build-push-action@v3
-        with:
-          tags: build:latest
-          context: .
-          target: build
-          load: true
-
-      - name: Run tests in the container locked with requirements_dev.txt
-        run: |
-          docker run --name test build bash /project/.github/workflows/container_tests.sh
```

Comment:
That's a shame, the container_tests.sh did an editable install, which bumped the coverage up to 100%. Without this test we aren't testing what the developer will be doing. I suggest we add an entry to the test matrix that runs with an editable install.

Reply:
Implemented the new matrix and coverage is back to 100%